inc/Engine/Optimization/RUCSS/Controller/UsedCSS.php

```diff
@@ -418,9 +417,25 @@ private function update_used_css( int $id, array $data ) {
 	 * @return string HTML content.
 	 */
 	private function remove_used_css_from_html( string $html, array $unprocessed_css ): string {
-		$html_nocomments    = $this->hide_comments( $html );
-		$link_styles        = $this->find( '<link\s+([^>]+[\s"\'])?href\s*=\s*[\'"]\s*?(?<url>[^\'"]+\.css(?:\?[^\'"]*)?)\s*?[\'"]([^>]+)?\/?>', $html_nocomments );
-		$inline_styles      = $this->find( '<style(?<atts>.*)>(?<content>.*)<\/style>', $html_nocomments );
+		$html_nocomments       = $this->hide_comments( $html );
+		$link_style_pattern    = '<link\s+([^>]+[\s"\'])?href\s*=\s*[\'"]\s*?(?<url>[^\'"]+\.css(?:\?[^\'"]*)?)\s*?[\'"]([^>]+)?\/?>';
+		$inline_inline_pattern = '<style(?<atts>.*)>(?<content>.*)<\/style>';
+
+		$link_styles   = $this->find(
+			'<noscript[^>]*>.*' .
+			'<link\s+([^>]+[\s"\'])?href\s*=\s*[\'"]\s*?([^\'"]+\.css(?:\?[^\'"]*)?)\s*?[\'"]([^>]+)?\/?>' .
+			'.*<\/noscript> (*SKIP)(*FAIL)|' .
```

Comment:
There is an extra space between noscript and SKIP `'.*<\/noscript> (*SKIP)(*FAIL)|' .` 
Will this cause any issues?